robot_sort/robot_sort.py

```diff
@@ -95,18 +97,48 @@ def light_is_on(self):
     def sort(self):
         """
         Sort the robot's list.
+        ###############################################
+        ------------------------Can Do----------------------------
+        * You may use any pre-defined robot methods.
+        * You may use logical operators. (`if`, `and`, `or`, `not`, etc.)
+        * You may use comparison operators. (`>`, `>=`, `<`, `<=`, `==`, `is`, etc.)
+        * You may use iterators. (`while`, `for`, `break`, `continue`)
+        * You may define robot helper methods, as long as they follow all the rules.
+
+        ----------------------Can Not Do------------------------
+        * You may NOT modify any pre-defined robot methods.
+        * You may NOT store any variables. (`=`)
+        * You may NOT access any instance variables directly. (`self._anything`)
+        * You may NOT use any Python libraries or class methods. (`sorted()`, etc.)
+        ################################################
         """
         # Fill this out
-        pass
+        while self.can_move_right():
```

Comment:
This is pretty incredible; I haven't seen an algorithm do it this way. I'll need to test this out on my own sometime! Very cool! :D